rllib_contrib/a3c/README.md

```diff
@@ -0,0 +1,50 @@
+# A3C (Asynchronous Advantage Actor-Critic)
+
+[A3C](https://arxiv.org/abs/1602.01783) is the asynchronous version of A2C, where gradients are computed on the workers directly after trajectory rollouts, and only then shipped to a central learner to accumulate these gradients on the central model. After the central model update, parameters are broadcast back to all workers. Similar to A2C, A3C scales to 16-32+ worker processes depending on the environment.
+
+
+## Installation
+
+```
+conda create -n rllib-a3c python=3.10
+conda activate rllib-a3c
+python -m pip install ray==2.3.1
+python -m pip install -e '.[development]'
+pre-commit install
+```
+
+## Usage
+
+```python
```

Comment:
Is there any way we can simply link into the below example script via sphinx tags?

Reply:
+1